VivistaServer/VivistaServer.csproj

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
     <PlatformTarget>x64</PlatformTarget>
     <Optimize>false</Optimize>
-    <DefineConstants>DEBUG;TRACE</DefineConstants>
+    <DefineConstants>DEBUG;TRACES</DefineConstants>
```

Comment:
Heb jij dit aangepast?

Reply:
Ik denk dat gij dit toen had aangepast om deels van de code uit te zetten met zo'n #IF

Reply:
Geen idee waarom ik TRACE zou hernoemen naar TRACES, maar bon, dan is het misschien door VS gedaan. Laat maar zo staan dan